base_test.go

```diff
@@ -1,3 +1,18 @@
+/*
+Copyright © 2020 - 2022 Dell Inc. or its subsidiaries. All Rights Reserved.
```

Comment:
Can you please remove the spaces between the years `2020-2022` . We have followed the same structure for the other repositories.

Thank you

Reply:
Updated all the file with changes you requested

Reply:
Thank you.